devito/compiler.py

```diff
@@ -86,7 +78,6 @@ def __init__(self, *args, **kwargs):
 
         if self.openmp:
             self.ldflags += ['-fopenmp']
-        self.pragma_ivdep = [Pragma('GCC ivdep')]
```

Comment:
I had to drop the ``GCC ivdep`` pragma because, unlike the ``ivdep`` pragma from Intel, it doesn't stack with the ``omp simd`` pragma, which **forces** loop vectorisation (ivdep is only a suggestion to skip dependency analysis, to speed up compilation time -- which for us is already extremely short with GCC).